app/router.ts

```diff
@@ -18,8 +19,13 @@ const Router = EmberRouter.extend({
         scheduleOnce('afterRender', this, () => {
             const page = this.get('url');
             const title = this.getWithDefault('currentRouteName', 'unknown');
+            const authenticated = this.get('session.isAuthenticated') ? 'Logged in' : 'Logged out';
+            const isPublic = 'true'; // This will have to be changed when we have private things
```

Comment:
We already have the dashboard page as a private thing--this should account for the dashboard.